lib/types/wadl.js

```diff
@@ -52,9 +52,14 @@ function convertToSwagger(wadl, callback) {
     }
   }
   var convertType = function(wadlType) {
-    assert(!_.isUndefined(wadlType));
-    assert(wadlType.indexOf('xsd:') === 0);
-    var type = wadlType.substring(4);
+    if (_.isUndefined(wadlType))
+      return {};
+
+    //HACK: we just strip namespace. Yes I know, it's ugly.
+    //But handling XML namespaces is even uglier.
+    var match = wadlType.match('^[^:]\+:(.+)$');
```

Comment:
Will there always be a namespace?

I.E. Should the regex be '^([^:]\+:)?(.+)$'